Puzzlepart/prosjektportalen365
# Issue/#759

### Your checklist for this pull request


### Description

Fix for issue #759. Fixed by rewriting the `ProjectTable` component to not use `DetailsList` from `office-ui-fabric-react`. The previously used `ListView` component from `@pnp/spfx-controls-react`, used `DetailsList` in the background. 

### How to test

1. Access a project site
2. On the right side of the home page, click the "Administrer underområder" (or english equivalent) button.
3. The rewritten ProjectTable component is the list and search box you see here.
4. The component is also used when you click on "Legg til område".

### Update of documentation
- [x] Update user manual if needed

### Relevant issues (if applicable)

#759 

💔Thank you!
